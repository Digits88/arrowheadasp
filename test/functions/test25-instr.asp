<!-- Test InStr function.
-->

InStr("", "hello") = 
	<%= InStr("", "hello") %><br>

InStr(NULL, "hello") = 
	<%= InStr(NULL, "hello") %><br>

InStr("blah", "") = 
	<%= InStr("blah", "") %><br>

InStr("blah", NULL) = 
	<%= InStr("blah", NULL) %><br>

InStr("thisissometext", "aba") = 
	<%= InStr("thisissometext", "aba") %><br>

InStr("thisisabanquettest", "aba") = 
	<%= InStr("thisisabanquettest", "aba") %><br>

InStr("thisisabanquettestabachacha", "aba") = 
	<%= InStr("thisisabanquettestabachacha", "aba") %><br>

InStr("quacksaidtheduck", "quack") = 
	<%= InStr("quacksaidtheduck", "quack") %><br>

InStr("quacksaidtheduck", "duck") = 
	<%= InStr("quacksaidtheduck", "duck") %><br>

InStr(1, "thisisabanquettestabachacha", "aba") = 
	<%= InStr(1, "thisisabanquettestabachacha", "aba") %><br>

InStr(6, "thisisabanquettestabachacha", "aba") = 
	<%= InStr(6, "thisisabanquettestabachacha", "aba") %><br>

InStr(7, "thisisabanquettestabachacha", "aba") = 
	<%= InStr(7, "thisisabanquettestabachacha", "aba") %><br>

InStr(8, "thisisabanquettestabachacha", "aba") = 
	<%= InStr(8, "thisisabanquettestabachacha", "aba") %><br>

InStr(1, "hErEiSaCaSeOfThEcAsEs", "seofth") =
	<%= InStr(1, "hErEiSaCaSeOfThEcAsEs", "seofth") %><br>

InStr(1, "hErEiSaCaSeOfThEcAsEs", "seofth", vbBinaryCompare) =
	<%= InStr(1, "hErEiSaCaSeOfThEcAsEs", "seofth", vbBinaryCompare) %><br>

InStr(1, "hErEiSaCaSeOfThEcAsEs", "seofth", vbTextCompare) =
	<%= InStr(1, "hErEiSaCaSeOfThEcAsEs", "seofth", vbTextCompare) %><br>

InStr(100, "thisisabanquettest", "aba") = 
	<%= InStr(100, "thisisabanquettest", "aba") %><br>
