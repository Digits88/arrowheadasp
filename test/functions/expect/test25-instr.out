<!-- Test InStr function.
-->

InStr("", "hello") = 
	0<br>

InStr(NULL, "hello") = 
	<br>

InStr("blah", "") = 
	1<br>

InStr("blah", NULL) = 
	<br>

InStr("thisissometext", "aba") = 
	0<br>

InStr("thisisabanquettest", "aba") = 
	7<br>

InStr("thisisabanquettestabachacha", "aba") = 
	7<br>

InStr("quacksaidtheduck", "quack") = 
	1<br>

InStr("quacksaidtheduck", "duck") = 
	13<br>

InStr(1, "thisisabanquettestabachacha", "aba") = 
	7<br>

InStr(6, "thisisabanquettestabachacha", "aba") = 
	7<br>

InStr(7, "thisisabanquettestabachacha", "aba") = 
	7<br>

InStr(8, "thisisabanquettestabachacha", "aba") = 
	19<br>

InStr(1, "hErEiSaCaSeOfThEcAsEs", "seofth") =
	0<br>

InStr(1, "hErEiSaCaSeOfThEcAsEs", "seofth", vbBinaryCompare) =
	0<br>

InStr(1, "hErEiSaCaSeOfThEcAsEs", "seofth", vbTextCompare) =
	10<br>

InStr(100, "thisisabanquettest", "aba") = 
	0<br>
